style.css

```diff
@@ -423,3 +380,197 @@ div .taggs {
+  }
+
+  #navbar {
+    padding: 0 6rem;
+  }
+
+  .navbar-menu {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    gap: 1rem;
+  }
+
+  .navbar-menu a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .navbar-image {
+    display: none;
+  }
+
+  .headline {
+    padding: 5rem 15rem;
+  }
+
+  .work-page-left {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+    margin: 1rem 0;
+    gap: 1rem;
+  }
+
+  .card2-content {
+    grid-row: 1/-2;
+    padding: 0;
+  }
+
+  .work-page-right {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+    gap: 1.5rem;
+  }
+
+  .clients {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding: 0 2px;
+    margin: 1px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  div .taggs {
+    float: right;
+    width: 40px;
+    height: 20px;
+    padding-bottom: 20px;
+  }
+
+  .tag1 {
+    width: 50px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    margin-bottom: 20px;
+    color: #396df2;
+  }
+
+  .tag2 {
+    width: 40px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    color: #396df2;
+  }
+
+  .tag3 {
+    width: 80px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    color: #396df2;
+  }
+
+  .about-myself {
+    width: 100%;
+    height: 496px;
+    display: flex;
+    align-items: center;
+    background: #fff;
+    padding: 5rem;
+    border-top-right-radius: 6rem;
+  }
+
+  .about-myself-content {
+    width: 50%;
+    float: left;
+    padding-right: 2rem;
+  }
+
+  .about-myself-languages {
+    width: 50%;
+    float: right;
+  }
+
+  .about-myself-title {
+    padding: 1rem 0;
+  }
+
+  .frameworks {
+    display: flex;
+    padding: 0.5rem 0;
+  }
+
+  .framework1 {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+  }
+
+  .framework-skills-desktop {
+    display: flex;
+  }
+
+  .html-skill,
+  .css-skill {
+    margin-right: 1rem;
+  }
+
+  .framework-skills {
+    display: none;
+  }
+
+  .contact {
+    display: flex;
+    justify-content: center;
+    background-image: url(images/desktop_main/contact_desktop.svg);
+    background-repeat: no-repeat;
+    background-position: left center;
+    background-size: contain;
+  }
+
+  .footer-content {
+    width: 30%;
+  }
+
+  .my-button {
+    margin: auto;
+    padding: 12px;
+    background: #fff;
+    border-radius: 8px;
+    color: #6070ff;
+    width: 50%;
+    border: none;
+    font-size: 17px;
+    font-weight: 500;
+  }

```

Comment:
- Good job done so far 👏. Please implement the hover effect styles for all your buttons. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Figma design:
<img width="338" alt="dmdm" src="https://user-images.githubusercontent.com/73607512/199808194-d3e2e3cf-ae92-47f5-b506-2c7a40212207.PNG">

Reply:
- I think adding a hover effect on all buttons in your portfolio will improve user experience and make it pretty. In order to do that, so can simply add this line of code below your `.project-detail ` style 👇🏽 

```
.project-detail:hover {
  background: #396df2;
  color: #fff;
}
```
It will be nice to implement this same hover effect on other buttons that you may have named differently. 🤗

Reply:
- [ ] Kindly as suggested by the previous reviewers, It would be better to implement the styling on your button interactions for a better user experience and to match the Figma design, :+1: :+1: 
as suggested before, you only need to and ```:hover``` to the class name to modify the stying on Hover state and these :point_down:  code can work for this, :joy: 
```
.project-detail:hover { 
  background: #396df2; 
  color: #fff; 
}
```

you can read the buttons interactions [here](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) for more details